x-pack/test_serverless/shared/services/deployment_agnostic_services.ts

```diff
@@ -8,12 +8,12 @@
 import _ from 'lodash';
 
 import { services as apiIntegrationServices } from '@kbn/test-suites-xpack/api_integration/services';
-import { AlertingApiProvider } from './alerting_api';
+import { services as apiIntegrationDeploymentAgnosticServices } from '@kbn/test-suites-xpack/api_integration/deployment_agnostic/services';
```

Comment:
Do we want load all the `deployment-agnostic` services or only `AlertingApiProvider` ?

Reply:
Glad you caught this. 

I took the risk when I did to see if it would pass all the tests.
That said, I do not mind merging it with less risk.

What do you think?

Reply:
Well, I can quickly see a conflict with the same service `sloApi` already [existing](https://github.com/elastic/kibana/blob/6848c6df33a562d1410dc80d72e9858ad5e814ab/x-pack/test_serverless/api_integration/services/index.ts#L39) in serverless.

It is really hard to find which service file was loaded, so I suggest explicitly picking only the ones that we need for particular PR. Ideally we can revisit it when more tests are migrated to deployment-agnostic format and "remove" duplicated services, but I would anyway keep importing explicitly 1 by 1 to speedup debug process

Reply:
plus folks might update `sloApi` in test_serverless, but FTR will still pickup deployment-agnostic version of it. Or the opposite, it depends on lines order :)

Reply:
yeah, that sounds like smth we should fix.  
This is not the PR for it, but constantly re-exporting `* from `abc` is prolly not the best for visibility.
If it's not great for us, it's gonna be worse for others

Reply:
https://github.com/elastic/kibana/pull/193975/commits/3f014027fbbf3902ce2b4bf167818d53b1d67316